scripts/pr-convergence.py

```
@@ -0,0 +1,971 @@
+            if after:
+                args.extend(["-F", f"after={after}"])
+            data = self.run(args)
+            if not isinstance(data, dict):
+                raise GhReadError("GraphQL response was not an object")
+            pull = (
+                data.get("data", {})
+                .get("repository", {})
+                .get("pullRequest")
+            )
+            if not isinstance(pull, dict):
+                raise GhReadError("GraphQL response did not include pullRequest")
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)